packages/forklift-console-plugin/src/modules/Providers/utils/validators/provider/openshift/openshiftSecretFieldValidator.ts

```diff
@@ -15,8 +15,11 @@ import { validateK8sToken, validatePublicCert, ValidationMsg } from '../../commo
  *
  * Validation msg and description should add information about the validation type.
  */
-export const openshiftSecretFieldValidator = (id: string, value: string): ValidationMsg => {
-  const trimmedValue = value.trim();
+export const openshiftSecretFieldValidator = (
+  id: string,
+  value: string | boolean,
```

Comment:
this does not look right, value can only be string (*now it's optional), where do you use boolean, what am i missing ? 
```suggestion
  value?: string,
```

Reply:
This was added for handling `insecureSkipVerify` field. 
I changed it now so that the `insecureSkipVerify`  field validation will be done for a string instead of a boolean.